### ISSUE TITLE
resize before load

### ISSUE THREAD

**Issue by ellenc345** [mspae/react-wavesurfer#29]
Hey, 
resizing before playing does not occurs.
When I tried to call the resize function it retrieved NaN because it does not have a position.

```
_secToPos(sec) {
    return (1 / this._wavesurfer.getDuration()) * sec;
  }
}
```
1/0 = NaN
It can be fixed by returning 0 when the player doesn't have a duration yet.

**Comment by omerts** [comment]
hey, has been fixed here: https://github.com/mspae/react-wavesurfer/pull/30

**Comment by omerts** [comment]
@ellenc345 This issue has been fixed, and released in v0.8.3.
